docs/leidraad/richtlijn-sjabloon-v1.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Sjabloon voor richtlijnen"
+tags: []
+---
+
+# Sjabloon voor richtlijnen, versie 1
+
+Vervang het kopje hierboven door "Richtlijn: _titel van de richtlijn_", bijvoorbeeld "Richtlijn: geautomatiseerd testen".
+
+Begin met een beschrijving van de richtlijn zonder deze te motiveren (dat komt in de volgende sectie).
+
+Leg termen uit en link deze (alleen bij de eerste keer dat ze gebruikt worden) naar openbare glossaries zoals de [Glossary of web terms](https://developer.mozilla.org/en-US/docs/Glossary) van het Mozilla Developer Network (MDN) of het [Online Browsing Platform](https://www.iso.org/obp/ui/en/#home) van de International Standards Organisation (ISO).
+
+Gebruik informele taal. Je kan de lezer aanspreken met "je". Afkortingen schrijf je de eerste keer uit met de afkorting tussen haakjes erachter.
+
+## Rationale: Waarom deze richtlijn?
+
+Vervang in het kopje hierboven "deze richtlijn" door de titel van de richtlijn.
+
+Motiveer waarom de richtlijn bijdraagt aan het realiseren van het principe waar deze onder valt. Noem ook risico's die optreden als de richtlijn niet wordt toegepast.
+
+## Doelgroep: Wie zijn er betrokken bij deze richtlijn?
+
+Vervang in het kopje hierboven "deze richtlijn" door de titel van de richtlijn.
+
+Beschrijf wat betrokken doelgroepen concreet met deze richtlijn kunnen doen. Introduceer eerst de doelgroepen met een zin in de vorm: "De volgende doelgroepen kunnen met geautomatiseerde tests aan de slag: developers en testers.". Vervolgens beschrijf je per doelgroep wat ze kunnen doen: "Developers kunnen...", "Testers beginnen met ...".
+
+## Implementatie: Hoe implementeer je deze richtlijn?
```

Comment:
Naast methoden en technieken en tools zijn er nog wel andere kopjes denkbaar. Opleidingen bijvoorbeeld. Maar dat kunnen we ook later toevoegen natuurlijk.

Reply:
OK, ik voeg opleiding alvast toe aan het sjabloon.